### ISSUE TITLE
About CustomNSURLProtocol something

### ISSUE THREAD

**Issue by zhangninghao** [rs/SDWebImage#2000]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I use CustomNSURLProtocol and sdwebimage 4.1.0 together in my app. In iOS 8.X.X something wrong happened

**Comment by dreampiggy** [comment]
@zhangninghao 
> Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

And also that `something wrong happened`, please provide detail information or stack trace for crash.

**Comment by bpoplauschi** [comment]
@zhangninghao I will close this issue because it lacks very important info. Feel free to reopen when you can properly fill the Issue Info table above and also describe the issue with details (not "something wrong happened").
